packages/unmock-core/src/__tests__/state.test.ts

```diff
@@ -272,7 +272,7 @@ describe("Test state management", () => {
 
   it("Handles textual state correctly", () => {
     const state = new State();
-    updateState(state, "any", "**", textMW("foobar"), "**");
+    updateState(state, "any", "**", textResponse("foobar"), "**");
```

Comment:
Not related to this PR, but this looks quite hard to read and error-prone with so many arguments...

Reply:
I understand, but it's a shortcut I made for internal tests (`updateState` is defined locally in this file as a shorthand for an otherwise vertically-breaking code).

Reply:
Sure, don't see though why internal helpers should not have the same standards as exposed functions :)

Reply:
Ah, just because in this case it loses it's function. I wanted to stop vertical line break, and making it e.g. a single object with parameters will again cause multiple line breaks.
If it bothers you I can revert back and eliminate that though.

Reply:
It doesn't bother me, just a comment.